conda-forge.yml

```diff
@@ -13,3 +13,7 @@ provider:
   linux_aarch64: default
   win: azure
 test: native_and_emulated
+noarch_platforms:
+  - linux_64
+  - osx_64
+  - win_64
```

Comment:
@hmaarrfk, do we need `win_64` here or can we remove it for now?

Other than that, this should be ready to go, right?

Reply:
i think we need it, otherwise it won't trick conda-smithy to generate the windows builds. maybe. i'm not sure, try without it?